utils/btrfs-progs/Makefile

```diff
@@ -72,8 +76,9 @@ define Package/btrfs-progs/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libbtrfs.so* $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libbtrfsutil.so* $(1)/usr/lib
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(addprefix $(PKG_INSTALL_DIR)/usr/bin/, $(progs)) $(1)/usr/bin/
-	$(LN) btrfs $(1)/usr/bin/btrfsck
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/btrfs.box $(1)/usr/bin/btrfs
+	$(foreach prog,$(boxprogs),$(LN) btrfs $(1)/usr/bin/$(prog);)
+	$(foreach prog,$(progs),$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(prog) $(1)/usr/bin/;)
```

Comment:
If you're not going to use the INSTALL_DIR anymore, might as well remove PKG_INSTALL.

Reply:
Tested. This is needed for the libraries.